### ISSUE TITLE
Minimal install for RC4 without middleware fails

### ISSUE THREAD

**Issue by ganeshkbhat** [zendframework/zend-expressive#230]
I am using the ZendExp rc4. Using (skeleton+installer) with options minimal option, fastroute, zendSM, NoView,Whoops. A Composer server or php dev server on it fails with 404 for the root 'http://localhost:8080/' path. Is it how it has to be - compulsory middleware need? Didnt see any docs on it. A full install option with other above configs work.

**Comment by xtreamwayz** [comment]
If you selected minimal (yes) at the first question, that is what it does. It doesn't add any actions, middleware, templates or routes. If you want something that works out of the box, you should install the full skeleton.

**Comment by weierophinney** [comment]
@ganeshkbhat — as @xtreamwayz notes, when you select the minimal install, it essentially only sets up the project structure, but without any middleware defined, or any default assets (CSS, JS, favicon, etc.). This is to allow experienced users to setup a project without immediately needing to remove those items and/or replace them with their own.

A default installation, *without* selecting "minimal", will install these things, as well as some sample middleware. That's the difference.

**Comment by ganeshkbhat** [comment]
Sure. Just downloaded the RC yesterday for creating APIs. Good to put in docs though. I was checking what went wrong until I started digging code internally. Thank you for coming back fast on this. Still digging into the docs and code. Quite some changes to previous zf2 skeleton. One more point, I think it is a good idea to put in docs that if Windows System users are not able to install using the installer ask them to run the CLI in administrator mode. Win systems seem to be blocking the download and setup without admin mode.

**Comment by xtreamwayz** [comment]
>  Win systems seem to be blocking the download and setup without admin mode.

Strange. I have developed the installer on a windows system an I have had no issues at all. Can you  create a new issue at https://github.com/zendframework/zend-expressive-skeleton/issues. Maybe add some more details so I can have a look at it.

**Comment by ganeshkbhat** [comment]
Yes, even I presume it could have been composer installation issue; may be rights sharing for my user. But I cannot find something like this; where composer.bat needed elevated rights. I may be wrong here. I am not able to replicate the issue as well. Lets wait on this. May be it might come up as I start working on projects and do a cache clean during that. If I am able to replicate that I will get in touch with you.
